### PR TITLE
Security fixes

### PR DESCRIPTION
**Pull request by twiss**

### CODE REVIEW

File: src/message.js [hunk 1]
```diff
@@ -181,23 +182,35 @@ Message.prototype.decryptSessionKeys = async function(privateKeys, passwords) {
...
-        if (!privateKeyPacket.isDecrypted()) {
-          throw new Error('Private key is not decrypted.');
-        }
-        try {
-          await keyPacket.decrypt(privateKeyPacket);
-          keyPackets.push(keyPacket);
-        } catch (err) {
-          util.print_debug_error(err);
+      await Promise.all(privateKeys.map(async function(privateKey) {
+        const primaryUser = await privateKey.getPrimaryUser(/* symEncryptedPacket.created, userId */);
```
Comment (sanjanarajan) [inline src/message.js]: What does this comment mean?

File: src/message.js [hunk 2]
```diff
@@ -181,23 +182,35 @@ Message.prototype.decryptSessionKeys = async function(privateKeys, passwords) {
...
+        let algos = [
+          enums.symmetric.aes256, // OpenPGP.js default
```
Comment (sanjanarajan) [inline src/message.js]: Should we replace this with the current config value? Or perhaps add that one as well?

Comment (twiss) [reply src/message.js]: Yeah, maybe. The issue is that if you change the config value, other people using the normal default will throw -- so maybe it's better to find that out quickly.
Similarly, GPG will warn even with our normal default if we send an AES256 message to someone without AES256 in the preferred keys - so maybe this is just an argument to switch to 3DES as the fallback, as the spec mandates, make that non-configurable, and then add the config value to the start of the preferred algos when generating keys (and use it for password encryption).

File: src/message.js (hunk 1 above)
Comment (sanjanarajan) [reply src/message.js]: Why can't we at least pass in the time though?

Comment (twiss) [reply src/message.js]: Yeah we can do that

Comment (twiss) [reply src/message.js]: Actually, it turns out the `symEncryptedPacket.created` property was wishful thinking, and it doesn't exist, so we don't have the time here.

File: src/message.js (hunk 2 above)
Comment (sanjanarajan) [reply src/message.js]: Yes, let's do that

**Pull request merged**